src/request/export/index.html

```diff
@@ -114,7 +114,7 @@ <h1 data-i18n="export-words-intro-heading" class="nq-h1">There is no password re
                                 <use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-paper-edit"/>
                             </svg>
                             <span data-i18n="recovery-words-intro-offline">
-                                Keep your Words offline, don’t enter them anywhere but in the Nimiq Safe.
+                                Keep your words offline, enter them nowhere but on keyguard.nimiq.com
```

Comment:
@julianbauer 
Suggestion to make the domain bold and add a period at the end:
```suggestion
                                Keep your words offline, enter them nowhere but on <strong>keyguard.nimiq.com</strong>.
```